### DISCUSSION TITLE
Aggiornamenti dopo la versione 1.5.8

### DISCUSSION THREAD

**Discussion by luigirotondi2009** [iisgiua/giuaschool#738] [Richieste di aiuto] [answered]
**Descrivi il problema**
Salve, facendo gli aggiornamenti dopo la versione 1.5.8, il docente come procede nell'inserire il voto finale del giudizio sospeso? 

**Indica come riprodurre l'errore**

**File di LOG**
Se necessario, allega il file di log (menu **SISTEMA > MANUTENZIONE**)

**Comment by trinko** [comment]
Questa non è una segnlazione di errore. 
La sposto nelle richieste di aiuto.

**Reply by luigirotondi2009** [reply]
Va bene, sa come aiutarci in questa mia domanda?

**Comment by trinko** [comment, marked as answer]
Lo scrutinio si svolge come tutti i precedenti:
- da amministratore, su **SCUOLA > SCRUTINI** si sceglie il periodo degli _Esami del giudizio sospeso_ e si imposta la data di inizio degli scrutini e di inserimento delle proposte di voto;
- i docenti delle materie con debito potranno inserire le proposte di voto da **LEZIONI > SCRUTINIO**
- è anche possibile (ma non obbligatorio) inserire un giudizio sulla prova svolta di recupero del debito (questo giudizio rimane agli atti nel registro, ma non viene inserito del verbale dello scrutinio);
- quando tutte le proposte sono inserite, il coordinatore (o un membro dello staff) apre lo scrutinio.

**Comment by trinko** [comment]
Attenzione: è fondamentale perché lo scrutinio possa essere svolto correttamente che nel registro siano stati svolti in precedenza gli scrutini finali.

**Reply by luigirotondi2009** [reply]
Abbiamo completato tutti gli scrutini per il nostro istituto, a breve ci saranno gli esami per gli alunni con giudizio sospeso.

**Reply by luigirotondi2009** [reply]
Dopo aver svolto gli scrutini per gli alunni con giudizio sospeso, si può procedere con il cambio d'anno?

**Reply by trinko** [reply]
> Dopo aver svolto gli scrutini per gli alunni con giudizio sospeso, si può procedere con il cambio d'anno?

Le procedure di archiviazione e cambio d'anno non sono state acora ricontrollate e c'è la possibilità che vadano in errore. Consiglio di aspettare.

**Reply by luigirotondi2009** [reply]
Ci sono novità sulla versione 1.5.9?

**Reply by trinko** [reply]
> Ci sono novità sulla versione 1.5.9?

Purtroppo ho poco tempo da dedicare allo sviluppo, per cui sarà rilasciata probabilmente a fine mese.
